eth/tracers/native/prestate.go

```diff
@@ -165,7 +166,11 @@ func (t *prestateTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64,
 	case stackLen >= 4 && op == vm.CREATE2:
 		offset := stackData[stackLen-2]
 		size := stackData[stackLen-3]
-		init := scope.Memory.GetCopy(int64(offset.Uint64()), int64(size.Uint64()))
+		init, err := tracers.GetMemoryCopyPadded(scope.Memory, int64(offset.Uint64()), int64(size.Uint64()))
+		if err != nil {
+			log.Warn("failed to copy CREATE2 input", "err", err, "tracer", "prestateTracer", "offset", offset, "size", size)
+			return
```

Comment:
@s1na thinking about this more personally I'd prefer this `panic(err)` instead of an early return, especially since the log warnings could get spammy under load since one line could be logged per RPC request triggering the issue.

Reply:
A `panic` in an rpc call would not crash the server, just the method-handler, and the user would get some form of error message like "method handler crashed", if I recall correctly. 

As I see it, we have these options for what to return to the user: 

1. A mostly correct trace + server-side printout indicating error
2. No trace + "method handler crashed" error
3. No trace + specific error about what happened and when ("failed to copy create2 input")

Not sure what my preference is. But previously, we settled for 1.